flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -1612,6 +1612,13 @@ class FMCMainDisplay extends BaseAirliners {
     setApproachIndex(approachIndex, callback = EmptyCallback.Boolean) {
         this.ensureCurrentFlightPlanIsTemporary(() => {
             this.flightPlanManager.setApproachIndex(approachIndex, () => {
+                const approach = this.flightPlanManager.getApproach();
+                if (approach) {
+                    const runway = this.flightPlanManager.getApproachRunway();
+                    if (runway) {
+                        SimVar.SetSimVarValue("L:A32NX_AUTO_LANDING_ELEVATION", "feet", runway.elevation * 3.28084);
```

Comment:
We should extract `3.28084` (which I assume is "meters to feet" ?) to a constant somewhere.

I suggest `flybywire-aircraft-a320-neo/html_ui/JS/A32NX_Util.js` for now, until we have the new FMGC in production, at which point we will refactor this anyway.

Reply:
Good call, I've actually created a function in `A32NX_Util` called `meterToFeet`, which should keep the code even cleaner (I hope) 😃.